src/webots/nodes/WbBaseNode.cpp

```diff
@@ -235,28 +236,70 @@ bool WbBaseNode::isInvisibleNode() const {
   return WbWorld::instance()->viewpoint()->getInvisibleNodes().contains(const_cast<WbBaseNode *>(this));
 }
 
+QString WbBaseNode::documentationUrl() const {
+  // TODO: Hotfix, undefined method
+  return QString();
+}
```

Comment:
I pulled this from `develop`, it didn't want to compile, so I have just defined it. I am not sure what to do with this since it is not related to this PR